### PR TITLE
Skip tests unless RefreshMode is Disabled

### PR DESCRIPTION
**Pull request by mcquin**
Remove implicit test assumption that the machine is configured so that `'RefreshMode'` is `'Disabled'` for DSC resource.

### CODE REVIEW

**Comment by jaym** [comment]
:+1:

**Comment by mcquin** [comment]
@chef/client-windows

File: lib/chef/platform/query_helpers.rb [hunk 1]
```diff
@@ -52,6 +52,13 @@ def supports_dsc_invoke_resource?(node)
           Gem::Version.new(node[:languages][:powershell][:version]) >=
             Gem::Version.new("5.0.10018.0")
       end
+      
+      def refresh_mode_disabled?(node)
```
Comment (chefsalim) [inline lib/chef/platform/query_helpers.rb]: should this have 'dsc' in the the function name.. ?

Comment (mcquin) [inline lib/chef/platform/query_helpers.rb]: yes, that would be useful

**Comment by chefsalim** [comment]
:+1:

**Pull request merged**